src/transformers/models/cpmant/configuration_cpmant.py

```diff
@@ -123,5 +120,4 @@ def __init__(
         self.eps = eps
         self.use_cache = use_cache
         self.vocab_size = vocab_size
-        self.return_dict = return_dict
```

Comment:
this could be initialized in super() with the correct default value, no need to add it here.
(in this case, keeping it will fail `utils/check_config_attributes.py`)